components/has/overlays/staging/kustomization.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base/argocd-roles
+  - ../../base/external-secrets
+  - https://github.com/redhat-appstudio/application-service/config/default?ref=a32c938ee1fcafaed4c64abd3ed4864121fd363c
+  - https://github.com/redhat-appstudio/application-service/config/prometheus/?ref=a32c938ee1fcafaed4c64abd3ed4864121fd363c
+
+components:
+  - ../../components/manager-resources
+  - ../../components/enable-profiling
+
+
+images:
+  - name: quay.io/redhat-appstudio/application-service
+    newName: quay.io/redhat-appstudio/application-service
+    newTag: 05ee33115e4cd7ebfad61d96ffe5ef36c8159dab
+
+configMapGenerator:
+  - literals:
+      - GITHUB_ORG="redhat-appstudio-appdata-staging"
+    name: github-config
+    behavior: replace
+
+patches:
+  - path: ./replicas_patch.yaml
```

Comment:
Can this be a component patch? Since staging and prod are using the same replica patch spec which is 3, they can be included as component here?

Reply:
yep, that's in the back of my mind too.  I think it's a good idea to move it to components